packages/util/package.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "@discordjs/util",
+	"version": "0.1.0",
+	"description": "Utilities shared across Discord.js packages",
+	"private": true,
+	"scripts": {
+		"build": "tsup",
+		"lint": "prettier --check . && TIMING=1 eslint src --ext mjs,js,ts",
+		"format": "prettier --write . && TIMING=1 eslint src --ext mjs,js,ts --fix",
+		"fmt": "yarn format"
+	},
+	"main": "./dist/index.js",
+	"module": "./dist/index.mjs",
+	"types": "./dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js",
+		"types": "./dist/index.d.ts"
+	},
+	"directories": {
+		"lib": "src"
+	},
+	"files": [
+		"dist"
+	],
+	"contributors": [
+		"Crawl <icrawltogo@gmail.com>"
+	],
```

Comment:
It's missing the rest of us:
https://github.com/discordjs/discord.js/blob/d08a57cadd9d69a734077cc1902d931ab10336db/packages/discord.js/package.json#L27-L33